deps/build.jl

```diff
@@ -1,15 +1,48 @@
 #######################################################################
-import JSON
+import JSON, Conda
 using Compat
 
+# remove deps.jl if it exists, in case build.jl fails
+isfile("deps.jl") && rm("deps.jl")
+
 # print to stderr, since that is where Pkg prints its messages
 eprintln(x...) = println(STDERR, x...)
 
 # Make sure Python uses UTF-8 output for Unicode paths
 ENV["PYTHONIOENCODING"] = "UTF-8"
 
-include("jupyter.jl")
-const jupyter, jupyter_vers = find_jupyter()
+
+const jupyter, jupyter_vers = try
+    "jupyter",convert(VersionNumber, chomp(readall(`jupyter-kernelspec --version`)))
+catch e1
+    try
+        "ipython",convert(VersionNumber, chomp(readall(`ipython --version`)))
```

Comment:
Hmm, same for me (MacOS with Anaconda).  @minrk, is this expected or is something wrong with Anaconda's Jupyter package?

In any case, I'm still not to worried as currently any `jupyter*` program should be fine (they all have versions > 3).